src/main/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorWorkflowStep.groovy

```diff
@@ -67,11 +67,24 @@ class IqPolicyEvaluatorWorkflowStep
     this.jobCredentialsId = jobCredentialsId
   }
 
+  @SuppressWarnings('Instanceof')
+  @DataBoundSetter
+  public void setIqApplication(final Object iqApplication) {
+    if (iqApplication instanceof IqApplication) {
+      this.iqApplication = iqApplication
+    }
+    else if (iqApplication instanceof String) {
+      this.iqApplication = new SelectedApplication(iqApplication)
```

Comment:
I believe this should be ManualApplication as if a String is provided the intended behavior is try to create a new application (as that is the case in the CLI)

Reply:
> I believe this should be ManualApplication as if a String is provided the intended behavior is try to create a new application (as that is the case in the CLI)

Selected or manual have the same outcome.  Either one calls the same method verifyOrCreate.  The old version of the GUI only had selected.  It was stored as a string.  The new version receives a string.  It defaults to selected.  The GUI matches what they had before.